### ISSUE TITLE
Array of valid values is using first value as default

### ISSUE THREAD

**Issue by pauloddr** [sebelga/gstore-node#66]
Hello. I have the following schema for a `Pose` model:

```javascript
{
  name: {type: 'string', required: true},
  skeleton: {values: ['HumanBody', 'HumanHand'], required: true},
  ...
}
```

When I try to create a new entity without `skeleton` attribute, it's automatically picking `HumanBody` as the default value:

```javascript
console.log(new Pose({name: 'Whatever'}))
// will output: {name: 'Whatever', skeleton: 'HumanBody'}
```

Thus, the `required: true` option has no effect because it's setting a default value.

This behavior is confusing, and also undesired in my case, as I need the attribute to be explicitly defined, raising a ValidationError when not given. How can I achieve this?

Thanks.

**Comment by sebelga** [comment]
Hello,

Yes indeed by design the default value is set to the first element of the values.  You are right that with the "required: true" setting it does not make sense. I will have a look into it.

Thanks for reporting!

**Comment by sebelga** [comment]
Fixed in release 1.3.1 Let me know if you still have the issue.

**Comment by pauloddr** [comment]
It worked. Thank you for the prompt response and update. :)
